fury/window.py

```diff
@@ -936,6 +936,11 @@ class ReportSnapshot(object):
     if colors is not None:
         if isinstance(colors, tuple):
             colors = [colors]
+        # Drop the alpha channel if the image/color objects do not have it
+        if len(colors[-1]) > im.shape[-1]:
+            colors = np.delete(colors, 3, axis=1)
+        if len(colors[-1]) < im.shape[-1]:
+            im = np.delete(im, 3, axis=1)
```

Comment:
After looking at [line 953](https://github.com/fury-gl/fury/pull/44/files#diff-1205ce6da451e0358e8b1b545a6cf719R953), I just wonder whether it will be better to do the same [line 947](https://github.com/fury-gl/fury/pull/44/files#diff-1205ce6da451e0358e8b1b545a6cf719R947) and remove this block? (`np.any(np.all(im[..., :3] == col[:3], axis=-1))`).
it will avoid you to add tests for all this case. What do you think @jhlegarreta?

Reply:
If this is an error, rather than the intended behavior, whether coming from miniconda or nibabel, we should may be check/report there? If the change in there was not intended, it may affect other projects, including DIPY/FURY themselves in some other parts (?).

In either case, and as for the aspect of adding more tests, I think the only case we are missing is the one the second `if` block addresses. I'd be happy to add a test if you spot some image dataset having RGBA.

If this is the new intended behavior, and as for the question itself, while writing the `if` blocks I too thought that it was too verbose, but could not think of a more elegant/compact way. Now you found it :thumbsup: . So using your proposed change seems reasonable, if, as said, we should not be making changes upstream.

Reply:
Sorry, it was not clear, I was just saying that Travis is failing because of Nibabel or Miniconda. It is not related to your changes. But like you said, DIPY is not affected so I believe that is Miniconda. I will create a PR to fix this issue tomorrow and you can rebase on that.

Concerning your PR, great that you like the other option 😄

Reply:
:thumbsup: OK, thanks for the explanation. I'll rebase 2761a29 once your PR gets merged then.